### ISSUE TITLE
No table_manager options in loki-simple-scalable deployment 

### ISSUE THREAD

**Issue by xeonic-ant** [grafana/loki#6830]
**Describe the bug**
No table_manager options in loki-simple-scalable helm deployment 
Also there is no information in documentation about this option: only about distributed and single-binary variants of deployment
To add table_manager options in config need to override it totally, because there are no paremeters in this block:
https://github.com/grafana/helm-charts/blob/main/charts/loki-simple-scalable/values.yaml#L62
I add:
```
table_manager:
  retention_deletes_enabled: true
  retention_period: 336h
```
i dont know is it working or not. I writer i see line like that:

```
level=info ts=2022-08-02T23:53:55.398039601Z caller=table_manager.go:167 msg="handing over indexes to shipper"
level=info ts=2022-08-02T23:54:55.396609083Z caller=table_manager.go:134 msg="uploading tables"
level=info ts=2022-08-02T23:54:55.397455092Z caller=table_manager.go:167 msg="handing over indexes to shipper"
level=info ts=2022-08-02T23:55:55.396612872Z caller=table_manager.go:134 msg="uploading tables"
level=info ts=2022-08-02T23:55:55.397687642Z caller=table_manager.go:167 msg="handing over indexes to shipper"
level=info ts=2022-08-02T23:56:55.396612243Z caller=table_manager.go:134 msg="uploading tables"
```

**To Reproduce**
1. Install loki in loki-simple-scalable variant
2. Add table_manager options to config file (override it totally)
3. Test is table_manager working or not

**Expected behavior**
Documentation explain where table manager located in loki-simple-scalable deployment, there are table_manager options in helm chart, and they works)

**Environment:**
 - Infrastructure: Kubernetes
 - Deployment tool: helm loki-simple-scalable

**Comment by openbob** [comment]
Same issue here. I thought that loki-simple-scalable is a good compromise for simple medium-size deployment. I was trying to set it up with s3 and dynamodb. The latter hit the issue as the chart does not seem to support table_manager.

**Comment by adamjohnson01** [comment]
We have the same issue.

**Comment by chjohnst** [comment]
Same issue here

**Comment by flixx** [comment]
Same issue here. I can also not find it in the [new helm chart](https://github.com/grafana/loki/tree/main/production/helm)

**Comment by JStickler** [comment]
Closed via #8098
